### DISCUSSION TITLE
Azure Trusted Signing and WiX Bootstrappers

### DISCUSSION THREAD

**Discussion by chrpai** [wixtoolset/issues#8836] [Questions]
v3 and v4/v5 so this is a votive and heatwave related question.

Has anyone used Azure Trusted Signing when building wix bootstrappers?  The github and azure devops actions (integrations) are pretty easy to use but it gets tricky when you have an MSI and EXE built as part of the same solution.  EXE trickier as you'd have to detach, sign and reattach.    

There is a way to call signtool passing it a bunch of additional parameters but then there is the question of how to install the needed packages and point to them.  I'm sure I'll figure it out but I was wondering if anyone already solved this in an easy elegant way?

**Comment by bn-pili** [comment]
We are facing the same issue now. Do you have a solution? Thank you!

**Comment by pepone** [comment]
I have used the pre-release sign package for this, It can be installed with "dotnet tool", see my workflow for VSIX package in 
https://github.com/zeroc-ice/ice-builder-visualstudio/blob/18e9238567cf47928450e44ec0407669037be528/.github/workflows/ci.yml#L27-L29

https://github.com/zeroc-ice/ice-builder-visualstudio/blob/18e9238567cf47928450e44ec0407669037be528/.github/workflows/ci.yml#L39
